deployment/cdk/opensearch-service-migration/lib/common-utilities.ts

```diff
@@ -435,3 +439,27 @@ export function isStackInGovCloud(stack: Stack): boolean {
 export function isRegionGovCloud(region: string): boolean {
     return region.startsWith('us-gov-');
 }
+
+/**
+ * Creates a Docker image asset from the specified image name.
+ *
+ * This function creates a Docker image asset from the specified image name.
+ * It can use a local or remote image.
+ *
+ * @param {string} imageName - The name of the Docker image to use as the base image.
+ * @returns {ContainerImage} - A `ContainerImage` object representing the Docker image asset.
+ */
+export function makeDockerImageAsset(scope: Construct, serviceName: string, imageName: string): ContainerImage {
+    const sanitizedImageName = imageName.replace(/[^a-zA-Z0-9-_]/g, '_');
+    const tempDir = mkdtempSync(join(tmpdir(), 'docker-build-' + sanitizedImageName));
+    const dockerfilePath = join(tempDir, 'Dockerfile');
+    const dockerfileContent = `
+        FROM ${imageName}
+    `;
+    writeFileSync(dockerfilePath, dockerfileContent);
+    const asset = new DockerImageAsset(scope, serviceName + 'Image', {
```

Comment:
appending Image for an image is a little bit weird.  I'd replace Image with something showing the purpose of why we're creating an aliased image.

Reply:
Renamed to just ServiceImage, this will be within each service stack